config/ut_jackal.lua

```diff
@@ -0,0 +1,81 @@
+RobotConfig = {
+  name = "ut-jackal";
+  scan_topic = "/velodyne_2dscan_high_beams";
+  pointcloud_topic = "";
+  odometry_topic = "/jackal_velocity_controller/odom";
+  initialpose_topic = "/initialpose";
+};
+
+enml = {
+  -- On the cobblestone next to the GDC door closest to the lab,
+  -- ********************************************************
+  -- DEFAULT STARTING LOCATION
+  -- On the cobblestone next to the GDC door closest to the lab,
+  -- right next to grass, on the side with the lamp post and facing Speedway.
```

Comment:
I agree that this config file probably isn't right place to put it, but this drops a lot of the commented out information for where the jackal started in various bag files.